server/routes/blockVisitDates/blockVisitDatesController.ts

```diff
@@ -46,18 +47,18 @@ export default class BlockVisitDatesController {
 
   public validate(): ValidationChain[] {
     return [
+      // store pre-sanitised user-entered value as rawDate to show in field validation error
+      body('rawDate').customSanitizer((_value, { req }) => (typeof req.body?.date === 'string' ? req.body.date : '')),
```

Comment:
With `express-validator`, once you've run a sanitisation function, you can't access the originally-entered input value. But need to show this back to user with validation messages (GDS error handling guidelines). So, bit of a hack, but setting original value as a property on `req.body` to keep hold of it.